src/main/java/com/uid2/shared/store/RotatingSaltProvider.java

```diff
@@ -121,7 +121,7 @@ public ISaltSnapshot getSnapshot(Instant asOf) {
             if (!snapshot.isEffective(asOf)) break;
             current = snapshot;
         }
-        return current;
+        return current != null ? current : snapshots.get(snapshots.size()-1);
```

Comment:
What if snapshots.size == 0? 
Spacing around the the -1
I would expect to see some test cases for this

Reply:
Added test and subsequently fixed a problem. 

We call loadContent() in operator's Main.java before getSnapshot is ever called so I think it should not be zero, unless our salt file really is empty.